src/math/tests/signed_integers_test.cairo

```diff
@@ -190,14 +190,14 @@ fn i9_test_div_rem() {
     let b = i9 { inner: 5_u8, sign: true };
     let (q, r) = i9_div_rem(a, b);
     assert(q.inner == 2_u8 && r.inner == 3_u8, '-13 // -5 = 2 r -3');
-    assert(q.sign == false && r.sign == true, '-13 // -5 -> positive');
+    assert(q.sign == false && r.sign, '-13 // -5 -> positive');
```

Comment:
```suggestion
    assert(!q.sign && r.sign, '-13 // -5 -> positive');
```

Reply:
prefer seing == false in that case, the "!" can be not noticed

Reply:
Hmm i don't think it's a problem and I don't think it's a good idea to encourage `== bool`

Reply:
Done everywhere then